modules/storage-persistent-storage-pv.adoc

```diff
@@ -141,6 +141,16 @@ the Pods that use these volumes are deleted.
 //|Local | ✅ | - |  -
 |===
 
+Red Hat OpenShift Container Storage 4.2 provides the following backends for these
+access modes:
+
+* ReadWriteOnce: ceph-rbd (block mode)
+* ReadWriteMany: cephfs (file mode)
+
+See
+link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.2/html-single/managing_openshift_container_storage/index#available-plug-ins_rhocs[Available dynamic provisioning plug-ins] 
+for more information.
+
```

Comment:
We already have OCS documented in our "Access modes" table beginning in 4.2 docs: https://docs.openshift.com/container-platform/4.2/storage/understanding-persistent-storage.html#pv-access-modes_understanding-persistent-storage

If helpful, we can add `ceph-rbd (block mode)` in the **RWO** column and `cephfs (file mode)` in the **RWX** column.

It looks like your feature branch (`ocs4.2-updates`) is checked out from OCP `master` so I'm not sure why the table in this file does not already show OCS in the "Volume Plug-in" column. (We added OCS as a supported plug-in starting in OCP 4.2.)

Also, I don't think we should be linking documentation from this section for OCS as then we would potentially need to include links for all other plug-ins. OCS content placement should make sense in the context of our docs, which linking doesn't here. (Similar to my comment above.)